style.css

```diff
@@ -632,3 +655,625 @@ input#Name {
+    width: 300px;
+    height: 100px;
+    line-height: 24px;
+    margin-top: -30px;
+    padding-left: 13px;
+  }
+
+  .Screen4D {
+    background-image: url(screen.png);
+    background-repeat: no-repeat;
+    display: flex;
+    margin-top: 0;
+    height: 388px;
+    background-size: cover;
+    width: 327px;
+    margin-left: 5px;
+  }
+
+  .PrintingData4 {
+    display: flex;
+    list-style: none;
+    width: 295px;
+    height: 81px;
+    margin-top: 143px;
+    padding-left: 13px;
+    padding-top: 30px;
+    line-height: 44px;
+    font-size: 30px;
+    font-style: normal;
+    font-family: "create Round", sans-serif;
+    color: #fff;
+  }
+
+  .textSegment4 {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-size: 14px;
+    font-weight: 400;
+    color: #fff;
+    width: 300px;
+    height: 100px;
+    line-height: 24px;
+    margin-top: -30px;
+    padding-left: 13px;
+  }
+
+  .Screen5D {
+    background-image: url(2D.png);
+    background-repeat: no-repeat;
+    display: flex;
+    margin-top: 0;
+    height: 388px;
+    width: 327px;
+    margin-left: 5px;
+  }
+
+  .Screen6D {
+    background-image: url(3D.png);
+    background-repeat: no-repeat;
+    display: flex;
+    margin-top: 0;
+    height: 388px;
+    width: 327px;
+    margin-left: 5px;
+  }
+
+  .secondProjectButtonD {
+    display: none;
+  }
+
+  /* about me page */
+  .AboutMePage {
+    width: 1440px;
+    height: 798px;
+    height: 1195px;
+    display: flex;
+  }
+
+  .Header {
+    margin-left: 142px;
+  }
+
+  .AboutMeTextSection {
+    display: none;
+  }
+
+  .AboutMeTextSection-D {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin-top: 24px;
+    margin-left: 142px;
+    display: flex;
+    width: 566px;
+    height: 148px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+  }
+
+  .ResumeButton-D {
+    margin-left: 144px;
+  }
+
+  .secondBackgroundImage-D {
+    width: 632px;
+    height: 373px;
+    background-position: center;
+    background-size: cover;
+    background-image: url(background-2D.png);
+    position: absolute;
+    top: 5%;
+    margin-top: 152px;
+  }
+
+  .AboutLine2 {
+    width: 85%;
+    top: 640px;
+    position: absolute;
+    border: 1px solid #fff;
+    align-items: center;
+    margin-left: 98px;
+    margin-right: 15px;
+  }
+
+  .MySkills {
+    margin-left: 0;
+    display: flex;
+  }
+
+  .containerTech {
+    margin-left: 124px;
+    display: grid;
+    grid-template-columns: repeat(3, 352px);
+    text-align: start;
+    margin-top: -58px;
+  }
+
+  .containerTech > :nth-child(3) {
+    margin-top: -4px;
+  }
+
+  .containerTech > :nth-child() {
+    margin-top: -3px;
+  }
+
+  .form {
+    flex-direction: row;
+    display: flex;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    max-width: 1440px;
+    height: 614px;
+  }
+
+  .smallPara {
+    text-align: start;
+    width: 373px;
+    margin-left: -40px;
```

Comment:
- [ ] Please fix the alignment of footer and form as form should be on top of footer and not side by side

<img width="1501" alt="Screenshot 2023-06-10 at 8 32 03 PM" src="https://github.com/bhushan354/Portfolio_Project/assets/44798044/0122cdd9-2eba-4ffe-b513-5696dd1ff3ea">

Reply:
fix alignments - done